web/src/utils.js

```diff
@@ -298,18 +295,16 @@ const hex = (value) => {
 
 /**
  * Converts an issue to a validation error
- * @function
  *
  * @todo This conversion will not be needed after adapting Section to directly work with issues.
  *
- * @param {import("~/client/mixins").Issue} issue
+ * @param {import("~/types/issues").Issue} issue
```

Comment:
I wonder if would be better to import types at the top of the file.

Reply:
Well, this is a JavaScript file.

Reply:
Ah, right! I overlooked that tiny detail :P